content/cpp/concepts/maps/terms/empty/empty.md

```diff
@@ -0,0 +1,97 @@
+---
+Title: 'empty'
+Description: 'Checks to see if an STL map object is empty (contains elements) or not and returns a boolean true or false value, respectively'
+Subjects: 
+  - 'Computer Science'
+Tags:
+  - 'Cpp'
+  - 'Data Structures'
+  - 'Documentation'
+  - 'Functions'
+  - 'Map'
+  - 'Programming'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
```

Comment:
```suggestion
---
Title: 'empty'
Description: 'Checks if the map is empty and returns true if it has no elements, otherwise it returns false.'
Subjects: 
  - 'Computer Science'
Tags:
  - 'Data Structures'
  - 'Documentation'
  - 'Functions'
  - 'Map'
  - 'Programming'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---
```